src/views/emails/template-parts/header/head/series-pass-styles.php

```diff
@@ -66,8 +66,13 @@
 		padding: 18px 0 0 0 !important;
 	}
 
+	.tec-tickets__email-table-content-upcoming-event-card__virtual {
+		font-size: 12px;
+		color: #141827;
```

Comment:
CSS props should be in alphabetical order.
```suggestion
		color: #141827;
		font-size: 12px;
```